x-pack/plugins/actions/server/builtin_action_types/servicenow.ts

```diff
@@ -89,6 +89,7 @@ export function getActionType({
 }): ActionType {
   return {
     id: '.servicenow',
+    minimumLicenseRequired: 'gold',
```

Comment:
@mikecote  @peterschretlen  I just spoke with @MikePaquette, can we make this `platinum`. There is a good chance that's where it will land

Reply:
Thanks, I went ahead and made it `platinum` here: https://github.com/elastic/kibana/pull/58668/commits/9669ebdecbc708782aa52790a659574e7066a807.